Makefile

```diff
@@ -35,7 +35,7 @@ node_modules: package.json yarn.lock ## Install node modules.
 
 gen-go: $(WIRE)
 	@echo "generate go files"
-	$(WIRE) gen -tags $(WIRE_TAGS) ./pkg/server
+	$(WIRE) gen -tags $(WIRE_TAGS) ./pkg/server ./pkg/cmd/grafana-cli/commands/runner
```

Comment:
Is there a reason that the runner is in the commands folder? If I am understanding correctly it isn't a command per-se but rather part of the grafana-cli infrastructure.

Reply:
No comment on that in particular here, just adding some some context, that `commands` package was created when I moved all of the logic out of `grafana-cli/main.go` so that it could be imported in enterprise.